sqlalchemy-stubs/orm/session.pyi

```diff
@@ -60,6 +64,9 @@ class Session(_SessionClassMethods):
     def bind_mapper(self, mapper, bind): ...
     def bind_table(self, table, bind): ...
     def get_bind(self, mapper: Optional[Any] = ..., clause: Optional[Any] = ...): ...
+    @overload()
+    def query(self, entity: Type[T], **kwargs) -> Query[T]: ...
+    @overload()
     def query(self, *entities, **kwargs): ...
```

Comment:
Can we make the second overload to return `Query[Any]`?

Reply:
I think we can even make it return `Query[Tuple]`, but I am not sure it's much more useful.